KsApi/mutations/mutation inputs/ChangeCurrencyInput.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
```

Comment:
Is this file a duplicate from the one located at `KsApi/models/ChangeCurrencyInput.swift`?

Reply:
It shouldn't be. I moved it to `KsApi/mutations/mutation inputs` because it looks like that's where we're keeping the inputs.